grl/utils/fs.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
```

Comment:
What is `fs.py`, filesystem? Let's get make things more explicit (as a general rule)